src/main/java/com/bootcamp/demo/model/BookScooter.java

```diff
@@ -0,0 +1,59 @@
+package com.bootcamp.demo.model;
+
+public class BookScooter {
```

Comment:
Usually, classes' names are nouns, rather than verbs.

Reply:
I'll change it to BookingScooter, it's ok?